### PR TITLE
Pre stage to stage

### PR DESCRIPTION
**Pull request by axelrod-blox**
- [x] create operator
- [x] update Metadata
- [x] change metadata values
- [x] remove metadata values
- [x] claim welcome page
- [x] not eligible screen
- [x] eligible screen
- [x] claim rewards
- [x] add token
- [x] view transaction link

### CODE REVIEW

**Review by meshin-dev — APPROVED** [review]

File: src/app/common/stores/applications/Distribution/Distribution.store.ts [hunk 1]
```diff
@@ -153,7 +157,13 @@ class DistributionStore extends BaseStore {
...
     const walletStore: WalletStore = this.getStore('Wallet');
     // eslint-disable-next-line radix
-    return walletStore.fromWei(parseInt(String(this.rewardAmount)).toString());
+    return Number(walletStore.fromWei(parseInt(String(this.rewardAmount)).toString())) - Number(this.claimedRewards);
+  }
+
+  @computed
+  get userEligibleRewards() {
+    // eslint-disable-next-line radix
+    return Number(this.userRewardAmount) + Number(this.claimedRewards);
```
Comment (alexstolr) [inline src/app/common/stores/applications/Distribution/Distribution.store.ts:166]: I think the Number here is redundant as this.userRewardAmount already returns a number.

File: src/app/common/stores/applications/Distribution/Wallet.store.ts [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 import DistributionStore from '~app/common/stores/applications/Distribution/Distribution.store';
+import {
+  changeCurrentNetwork, getCurrentNetwork,
+  inNetworks,
+  NETWORKS, notIncludeMainnet, testNets,
+} from '~lib/utils/envHelper';
+
+const WALLET_CONNECTED = 'WalletConnected';
+
+const TOKEN_NAMES = {
```
Comment (alexstolr) [inline src/app/common/stores/applications/Distribution/Wallet.store.ts:21]: Can these be defined in a configuration rather then in the code itself?

**Review by alexstolr — APPROVED** [review]
Went over it but I am not familiar with the flows and also with FE. added a couple of comments and approved due to already existing approvals.
I see we have a lot of checks of which env and chain we are currently on. Maybe we should consider moving these to a configuration file and making the code more general.

File: src/app/common/stores/applications/Distribution/Wallet.store.ts (hunk 2 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/Distribution/Wallet.store.ts]: done

File: src/app/common/stores/applications/Distribution/Distribution.store.ts (hunk 1 above)
Comment (axelrod-blox) [reply src/app/common/stores/applications/Distribution/Distribution.store.ts]: done

**Pull request merged**